docs/openapi/components/schemas/credentials/DeMinimisExemptionCertificate.yml

```diff
@@ -0,0 +1,136 @@
+$linkedData:
+  term: DeMinimisExemptionCertificate
+  '@id': https://w3id.org/traceability#DeMinimisExemptionCertificate
+title: De Minimis Exemption Certificate
+description: Ecommerce import declaration under Tarif Act Section 321 amendment, providing for an administrative de minimis exemption from duty and taxes for shipments of merchandise (https://www.govinfo.gov/content/pkg/FR-2019-07-23/pdf/2019-15625.pdf).
+type: object
+properties:
+  '@context':
+    type: array
+    items:
+      type: string
+      const:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    items:
+      type: string
+      const:
+        - VerifiableCredential
+        - DeMinimisExemptionCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    $ref: ../common/DeMinimisExemption.yml
+  proof:
+    type: object
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:key:z6MkrPPZ84figsK3ksWXntFKJGANiJWJYwcusLc9tEKEmgsF",
```

Comment:
stop doing this :)

Reply:
use `urn:uuid:1231231` etc...

Reply:
This is the `issuer.id`, the document needs its own id

Reply:
"This" is the credential `id`... it should not be a DID, that is confusing.

Reply:
> "This" is the credential id... it should not be a DID, that is confusing.

Well... It SHOULD (dare I say, MUST?) be a URI, among which DIDs are found, so it *MAY* be a DID, and making it so *MAY* help decrease confusion in the wild though it MAY increase confusion during early adoption and other learning phases.

Reply:
Happy to change it, if nothing else to make you happy @OR13. :)

Reply:
Thank you.